core/spec/models/spree/stock/location_filter/active_spec.rb

```diff
@@ -13,8 +13,10 @@ module LocationFilter
 
         it 'returns only active stock locations' do
           active_stock_locations = instance_double('Spree::StockLocation::ActiveRecord_Relation')
-          expect(stock_locations).to receive(:active) { active_stock_locations }
 
+          without_partial_double_verification do
+            expect(stock_locations).to receive(:active) { active_stock_locations }
+          end
```

Comment:
Hmm, I am not a huge fan of these kind of specs. Could we rewrite the spec into something that actually test the result instead of stubbing the implementation? 

Create two stock locations, one active, one not. Expect the active one is the result.